config/initializers/locale.rb

```diff
@@ -0,0 +1,4 @@
+I18n.load_path += Dir[Rails.root.join('lib', 'locale', '*.{rb,yml}')]
+
+I18n.available_locales = [:en, :ja]
+I18n.default_locale = :ja
```

Comment:
こっちに設定書くなら `application.rb`側は不要ですね。日本語の説明わかりにくいので原文のほうが正しく理解できるかもです。

https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module

> The load path must be specified before any translations are looked up. To change the default locale from an initializer instead of config/application.rb:

あたりですね。

Reply:
なるほどです〜！サイトや本やらを参考にしていろんなところに書いてしまって必要なのか必要じゃないのかが分からなくなっていました。

Reply:
個人が書いた各種サイトの情報とかって結構いい加減なところありますから混乱しますよねー